misc/aws-k8s-cni-calico.yaml

```diff
@@ -79,12 +83,6 @@ spec:
     spec:
       hostNetwork: true
       serviceAccountName: calico-node
```

Comment:
Does calico-node still need access to the datastore when using typha?
Right now calico-node and typha use the same serviceAccount, any reason to give them separate ones.

Reply:
Yeah, they maybe could have different ones, but calico/node still needs data store access.